dist/tsc/sponsorclient.d.ts

```diff
@@ -0,0 +1,46 @@
+import { ethers, FetchRequest, JsonRpcApiProviderOptions, Networkish } from 'ethers';
+import type { AddressLike } from 'ethers/src.ts/address';
+import type { BigNumberish } from 'ethers/src.ts/utils';
+export declare enum WhitelistType {
```

Comment:
please put some comment on each field. you can copy from our doc.

Reply:
dist/ is auto generated